package.json

```diff
@@ -32,7 +32,7 @@
     "yup": "^0.32.11"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "set PORT=3069 && react-scripts start",
```

Comment:
a constant port is required as credentials cannot be included with CORS header set to '*'